### ISSUE TITLE
rear missing genisoimage dependency (on Fedora 26)

### ISSUE THREAD

**Issue by gdha** [rear/rear#1192]
* rear version (/usr/sbin/rear -V): rear-2.00-1.fc25.x86_64
* OS version (cat /etc/rear/os.conf or lsb_release -a): f25
* rear configuration files (cat /etc/rear/site.conf or cat /etc/rear/local.conf):
* Are you using legacy BIOS of UEFI boot? BIOS
* Brief description of the issue: Running "rear mkrescue" it fails because on the system is missing mkisofs, part of genisoimage.

1. install rear
2. create rescue media with "rear mkrescue"
3. ERROR: Could not find 'mkisofs' compatible program. Please install
'mkisofs', 'genisoimage' or 'ebiso'...

Actual results:
Rescue media not create

Expected results:
Media creation

* Work-around, if any:

````
--- rear.spec	2017-02-06 12:36:20.193617522 +0100
+++ rear.spec.2	2017-02-06 12:36:53.247407444 +0100
@@ -92,7 +92,7 @@
 %if %{?centos_version:1}%{?fedora_version:1}%{?rhel_version:1}0
 Requires: crontabs
 Requires: iproute
-Requires: mkisofs
+Requires: genisoimage
 #Requires: redhat-lsb
 %endif
````

Original bug report is [RedHat Bugzilla 1419512](https://bugzilla.redhat.com/show_bug.cgi?id=1419512)

**Comment by gozora** [comment]
Hello @gdha,

I think this is kind of similar problem as https://github.com/rear/rear/issues/1193. 
I'd say that _genisoimage_ is mandatory only for _OUTPUT=ISO_. So if user decides to use e.g. _OUTPUT=PXE_ _genisoimage_ (nor any other ISO generating program) is not needed any more ...

V.

**Comment by gdha** [comment]
@gozora Indeed that is correct, however, that being said `OUTPUT_ISO` is the default setting within ReaR. Therefore, it makes sense to add this one to the Required rpm list within our package.

**Comment by jsmeix** [comment]
I think because this issue is marked as "fixed" it can be closed.

**Comment by gdha** [comment]
According BZ#1419512 this is not fixed in rear-2.2 I will re-open this for further investigation

**Comment by gdha** [comment]
Found the solution at https://en.opensuse.org/openSUSE:Build_Service_cross_distribution_howto 
**Note that Fedora uses "%{fedora}" without "_version".**
